src/SDKs/_metadata/dns_resource-manager.txt

```diff
@@ -1,10 +1,11 @@
-2017-10-12 22:15:47 UTC
+2018-03-08 22:10:22 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      current
-Commit:      765c5a6807e3a093868bff793ec3e8ea0cdabe75
+GitHub user: owolabi2
+Branch:      koowolab/private-dns-public-preview
```

Comment:
@muwaqar this SDK will need to be generated from master of spec repo.

Reply:
Will do once our Swagger PR (https://github.com/Azure/azure-rest-api-specs/pull/2587) is approved.